<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dg_OutSetting</class>
 <widget class="QDialog" name="Dg_OutSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wg_outsetting" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="pic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>659</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> 

</string>
      </property>
      <property name="text">
       <string>  小条打印输出配置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>11</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/home/home/guanbi.png);
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame_outsetting">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>621</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="cbb_sex_outset">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_age_outset">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>年龄</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_name_outset">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_doctor_outset">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>检验医师</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_addr_outset">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>检验地址</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_class_outset">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>科室</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_classnum_outset">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>住院号</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_testtype_outset">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>样品类型</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_typeman_outset">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>患者类型</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbb_testnum_outset">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>样品编号</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbb_bed_outset">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>床 号</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_checkman_outset">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>审核者</string>
       </property>
      </widget>
     </widget>
     <widget class="MainButton" name="bt_ok">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>340</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="MainButton" name="bt_cancal">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
